python/lsst/pex/config/listField.py

```diff
@@ -499,11 +501,9 @@ def _compare(self, instance1, instance2, shortcut, rtol, atol, output):
         name = getComparisonName(
             _joinNamePath(instance1._name, self.name), _joinNamePath(instance2._name, self.name)
         )
-        if not compareScalars(f"isnone for {name}", l1 is None, l2 is None, output=output):
-            return False
-        if l1 is None and l2 is None:
-            return True
-        if not compareScalars(f"size for {name}", len(l1), len(l2), output=output):
+        if l1 is None or l2 is None:
+            return compareScalars(f"{name}", l1, l2, output=output)
```

Comment:
This does make me wonder whether `str(name)` is faster than `f"{name}"`

Reply:
I think this is actually guaranteed to be string already, so I'll replace it with `name`; I think it was originally a longer f-string.